### PR TITLE
Upgrade to Symfony 5.3

### PR DESCRIPTION
**Pull request by HedicGuibert**
Upgrades the project to Symfony 5.3.
I also required phpunit in its latest version.

An unexpected behaviour happened however : the font used for the titles changed. I found it to be fine, but I could revert it to the way it used to be.

Before : 
before

After :
after

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,4 @@
+###> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=a6c768fc6d9d978d22d25530015a5325
+APP_SECRET=
```
Comment (lyrixx) [inline .env:3]: This is suspect

File: composer.json [hunk 2]
```diff
@@ -16,28 +16,31 @@
...
-        "symfony/form": "5.1.*",
-        "symfony/framework-bundle": "5.1.*",
+        "symfony/form": "5.3.*",
+        "symfony/framework-bundle": "5.3.*",
+        "symfony/intl": "5.3.*",
         "symfony/monolog-bundle": "^3.0",
-        "symfony/phpunit-bridge": "^5.1",
-        "symfony/twig-bundle": "5.1.*",
-        "symfony/validator": "5.1.*",
+        "symfony/phpunit-bridge": "^5.3",
```
Comment (lyrixx) [inline composer.json:28]: This should be in require-dev, and phpunit should be removed

File: phpunit.xml.dist.bak [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lyrixx) [inline phpunit.xml.dist.bak:1]: File to remove

**Review by lyrixx — COMMENTED** [review]
Looks good to me, but tests are broken

**Comment by lyrixx** [comment]
ping @damienalexandre for the font (see PR description)

**Review by damienalexandre — COMMENTED** [review]
Could you investigate? Never seen this font :thinking: 
Font Familly is `font-family: Courgette;` but my browser fallback to `DejaVu Serif`. Was it never properly installed maybe?

**Comment by HedicGuibert** [comment]
It seems like it because I checked the font on several sites and it looked like the new one, so yes it was never really installed

File: phpunit.xml.dist.bak (hunk 3 above)
Comment (lyrixx) [reply phpunit.xml.dist.bak:1]: ping @HedicGuibert

**Comment by lyrixx** [comment]
@damienalexandre Could we keep the new font, or should be rollback to the old one?
It's the only missing piece to merge this PR 👍🏼

**Review by damienalexandre — APPROVED** [review]
ok :+1: thx !

**Comment by lyrixx** [comment]
Thanks @HedicGuibert

**Pull request merged**